pytorch_lightning/callbacks/quantization.py

```diff
@@ -114,6 +117,10 @@ def __init__(
             observer_type: allows switching between ``MovingAverageMinMaxObserver`` as "average" (default)
                 and ``HistogramObserver`` as "histogram" which is more computationally expensive.
 
+            max_num_inputs: maximum number of inputs nodes used by the model in training, evaluation, or
+                testing. This determines how many ``QuantStubs`` are needed. If not specified, the model's
+                ``example_input_array`` property must be defined
```

Comment:
might need some adjusting

```suggestion
                :attr:~pytorch_lightning.core.lightning.LightningModule.example_input_array` property must be defined
```